pybossa/core.py

```diff
@@ -118,6 +122,27 @@ def configure_app(app):
     app.url_map.strict_slashes = app.config.get('STRICT_SLASHES')
 
 
+def setup_request_profiler(app):
+    @app.before_request
+    def start_yappi():
+        if request.args.get('profileit') == '1':
```

Comment:
I'm wondering if a less verbose url argument would be beneficial. Perhaps just, `p=1` in place of `profileit=1`?

Reply:
ah, I made it verbose on purpose so I know there are no conflicts with other parameters!